### DISCUSSION TITLE
Feature Idea: add apfollow to LittleLink

### DISCUSSION THREAD

**Discussion by jaschaurbach** [JulianPrieber/littlelink-custom#203] [Feature requests]
I stumbled over https://github.com/mwt/apfollow and gave it a try at my website. 

It lets follow users on different ActivityPub platform like mastodon, pixelated or pleroma by just entering your own Fediverse-ID. In fact, it saves you some klicks.

How does it work?

The URL https://URL TO APFOLLOW/?user=herrurbach&instance=chaos.social produces a form where you enter your Mastodon-ID and hit "proceed" to follow int this case "@herrurbach@chaos.social".

It works the same for any other AP-Service which supports web finger and remote following (the tool does check this).

**Comment by JulianPrieber** [comment]
We are currently working on the big frontend update. 

In the course of this, we are reworking the "add link" page. We are implementing a new system that treats specialty buttons like the title and space element as separate modules.

After this update, we want to expand the list of available modules.
I'll try to implement this system as one of the new modules. If everything goes well, I'll include it. 👍
